README.markdown

```diff
@@ -293,14 +392,41 @@ Use implicitly unwrapped types declared with `!` only for instance variables tha
 When accessing an optional value, use optional chaining if the value is only accessed once or if there are many optionals in the chain:
 
 ```swift
-myOptional?.anotherOne?.optionalView?.setNeedsDisplay()
+self.textContainer?.textLabel?.setNeedsDisplay()
```

Comment:
Please remove **self** from here. Self references are done only when needed as described in the **Use of Self** section.